### PR TITLE
Tweaks for Analog Clock

### PR DESCRIPTION
**Pull request by AlexxBoo**
Suggested changes to the  antialiasPixelAR function to reflect the dAngle parameter. (Try larger values for the handWidths for some nice fat&blurry hands. 😊)
Further fixed the 0° hand-skipping: The hands tended to skip at 12 o'clock since the alias effect did not wrap around.
Also the static angle variables are now initialized with the actual derived values, which prevents them to be initially show at 0°.

P.S.
Is it intentional that the timeClient.update(); call in the main loop() is commented out?

### CODE REVIEW

File: Map.h [hunk 1]
```diff
@@ -78,20 +78,17 @@ void andPixelAR(uint8_t angle, uint8_t dAngle, uint8_t startRadius, uint8_t endR
...
-
   for (uint16_t i = 0; i < NUM_LEDS; i++) {
     uint8_t o = i;
 
     uint8_t ao = angles[o];
 
-    uint8_t adiff = qsub8(max(ao, angle), min(ao, angle));
-    uint8_t fade = qmul8(adiff, 32);
+    uint8_t adiff = min(sub8(ao,angle), sub8(angle, ao));
+    uint8_t fade = map(adiff, 0, dAngle, 0, 255);
```
Comment (jasoncoon) [inline Map.h:87]: I think map8 might be faster here, for mapping from one uint8_t range to another.

**Comment by jasoncoon** [comment]
I commented out `timeClient.update()` while troubleshooting periodic "glitches", and left it commented out for those not using clock functionality.  I'll probably re-enable it, with a higher update interval, and wrap it in a "enable clock" variable/const check.

File: Map.h (hunk 1 above)
Comment (AlexxBoo) [reply Map.h:87]: unfortunately the FastLED map8 function yields not the expected result
_map8: map from one full-range 8-bit value into a narrower range of 8-bit values_
In this case the incoming value is limited (by dAngle) and needs to be mapped to a full range. I found no equivalent function in the FastLED math library (also the lerp function don't help, they all expect a full range input value, i.e. "fraction".)

Comment (jasoncoon) [reply Map.h:87]: Ah, sorry, you're right: http://fastled.io/docs/3.1/group__lib8tion.html#gaf65357c7aeca9778e5a2f80a6bac2b5d

```
map8: map from one full-range 8-bit value into a narrower range of 8-bit values, possibly a range of hues.
```

**Pull request merged**